tests/acceptance/customCommands/uploadRemote.js

```diff
@@ -39,7 +39,7 @@ class UploadRemote extends events.EventEmitter {
           this.client.transport
             .runProtocolAction(opt) // private api
             .then(result => {
-              if (result.status !== 0) {
+              if (Object.hasOwnProperty.call(result, 'status') && result.status !== 0) {
```

Comment:
What's the reason for this change? Strict check on int 0 should already be sufficient, isn't it? Or can the result be undefined? In that case `result?.status !== 0` should do the same.

Reply:
on my local machine the tests had result.status undefined here and there but the result.value was ok

Reply:
But still, if `result.status` evaluates to `undefined` then it's also `!== 0`. No need for an additional check.

Reply:
if the result.value is ok but the result object has no property status it fails... this fixes this. Or do i misunderstand it? 

``` javascript
  // should pass
  { value: 'ok' }

  // should pass
  { value: 'ok', status: 0 }

  // shouldn't pass
  { value: 'wrong', status: 1 }
```

Reply:
Ah. Sorry. Not enough coffee, yet.